js/interface.js

```diff
@@ -513,14 +513,14 @@ Fliplet.Pages.get()
       }
 
       $select.append(
-        '<option value="' + page.id + '"' +
-        (widgetInstanceData.page === page.id.toString() ? ' selected' : '') +
-        '>' + page.title + '</option>'
+        '<option value="' + page.id + '"'
+        + (widgetInstanceData.page === page.id.toString() ? ' selected' : '')
+        + '>' + page.title + '</option>'
```

Comment:
[MINOR] @tonytlwu, that should be a template string

Reply:
Updated